client/view/Footer/Footer.css

```diff
@@ -1,12 +1,7 @@
 .Footer {
   display: flex;
   align-items: center;
-  padding: 12px 32px 24px;
+  width: var(--block-width);
+  padding: var(--block-padding);
   margin: 0;
```

Comment:
Let's make these changes:

1. Add `box-sizing: border-box`. To make the `.Footer` width equal to the `.Article`
2. Remove unused `display: flex` and `align-items: center` for `.Footer`
3. Set `display: inline-block` to `.Badge`

Reply:
Done it here
 https://github.com/browserslist/browsersl.ist/pull/463